community/kernel/src/main/java/org/neo4j/kernel/impl/api/KernelStatement.java

```diff
@@ -338,6 +349,10 @@ private void recordOpenCloseMethods()
 
         private static String buildMessage( Deque<StackTraceElement[]> openCloseTraces )
         {
+            if ( openCloseTraces.isEmpty() )
```

Comment:
Could check `openCloseTraces != null || openCloseTraces.isEmpty()` here and removed `EMPTY_STATEMENT_HISTORY`.